components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -3382,7 +3386,7 @@ def definition_body(self):
 """ % proto_properties))
 
         if self.descriptor.interface.hasInterfaceObject():
-            properties["name"] = str_to_const_array(name)
```

Comment:
This would require change in https://github.com/servo/servo/blob/1c6b74e1f16f6c92f80410d98608064fa4fbede2/components/script/dom/bindings/interface.rs#L278-L290